directories_and_file_paths/directories_and_file_paths.md

```diff
@@ -0,0 +1,193 @@
+<!--
+
+author:   Meredith Lee
+email:    leemc@chop.edu
+version:  1.0.0
+language: en
+narrator: UK English Female
+title: Directories and File Paths
+comment: In this module, learners will explore what a directory is and how to describe the location of a file using its file path.   
+long_description: When doing data analysis in a programming language like R or Python, figuring out how to point the program to the file you need can be confusing. This module will help you learn about how files and folders are organized on your computer, how to describe the location of your file in a couple of different ways, and name files and folders in a descriptive and systematic way.
+@learning_objectives  
+
+After completion of this module, learners will be able to:
```

Comment:
Objectives 2 and 4 are covered in quiz questions, but I don't feel like 1 and 3 are yet. For 1, you could do something like "which of the following are synonyms for "directory"? drive, volume, folder, file" to make sure they're correctly disambiguating the terms you're using. For 3, maybe a write-in question saying something like "Write out your best guess for the relative file path for a file called "data.csv" in a subfolder called "raw_data" within a project folder called "dissertation", when "dissertation" is the current working directory"?

Reply:
I think these are great, they're going in :)